system/k3s/k3s.SlackBuild

```diff
@@ -60,6 +60,21 @@ else
   exit 1
 fi
 
+bailout() {
+  echo "  Better to have a k3s group to run this script. "
+  echo "  create k3s group and add user to avoid always use of sudo: "
+  echo "    # groupadd k3s "
```

Comment:
You need to request a new gid - they are kept here to avoid conflicts across sbo packages:

https://slackbuilds.org/uid_gid.txt

Reply:
I'll remove bailout for now